packages/wrangler/src/dev/miniflare.ts

```diff
@@ -985,6 +985,14 @@ export async function buildMiniflareOptions(
 
 	if (config.bindings.images && config.imagesLocalMode) {
 		if (!didWarnImagesLocalModeUsage) {
+			try {
+				await import("sharp");
+			} catch {
+				const msg =
+					"Sharp must be installed to use the Images binding local mode; check you are using Node >= 18";
```

Comment:
It looks like sharp doesn't support Node 19 either (see https://sharp.pixelplumbing.com/install#prerequisites). How about suggesting the users to check Node compatibility on the Sharp website?